### ISSUE TITLE
Italian catalog problem

### ISSUE THREAD

**Issue by samo79** [AmiBlitz/AmiBlitz3#15]
The italian translation in PED seems doesn't work correctly, catalog are pick but the strings are placed in a wrong order, this make the use of italian almost unusable.
Please check it

**Comment by samo79** [comment]
Checked a bit PED.cd and noted that it was completely changed compared to the one i was worked on ... This will force me (and others) to almost rewrite it again..
Please let us know when you will have a stable version of PED.cd, also if you have to add new things in catalog add them at the end of the file, and never in the middle, because this will make our catalogs completely unusable (strings will be placed in a wrong order!)

Using FlexCat to compile i've also found  that a string was used twice, this one:

MSG_ARGS_70_NAME

**Comment by honitos** [comment]
Yes of course. I am sorry for that. I am not very familiar with localization or the mechanism of it.
I will need help after rewrite of PED is done.

**Comment by polluks2** [comment]
Although you may increment the catalog's version. Better append new messages add the end of the catalog to maintain compatibility.

**Comment by Hexaae** [comment]
Italian has also an issue with About requester showing wrong strings (same for French):
![image](https://user-images.githubusercontent.com/8591462/165967030-ff07e2d3-1a22-424d-a287-b93366f64bc3.png)

**Comment by polluks2** [comment]
Well, the French catalog is still version 3.8. I suppose it's not updated.

**Comment by samo790** [comment]
Problem still

**Comment by samo790** [comment]
Any news about this long standing issue?
I would love to use italian and of course update the translation but at the moment is not usable in this state.
As Pollucks says to avoid mess better adding new strings always at the end of the catalog in order to maintain compatibility
